cleanlab/object_detection/summary.py

```diff
@@ -34,6 +42,184 @@
+        This is a list of ``N`` ``np.ndarray`` such that ``predictions[i]`` corresponds to the model prediction for the `i`-th image.
+        For each possible class ``k`` in 0, 1, ..., K-1: ``predictions[i][k]`` is a ``np.ndarray`` of shape ``(M,5)``,
+        where ``M`` is the number of predicted bounding boxes for class ``k``. Here the five columns correspond to ``[x1,y1,x2,y2,pred_prob]``,
+        where ``[x1,y1,x2,y2]`` are coordinates of the bounding box predicted by the model
+        and ``pred_prob`` is the model's confidence in the predicted class label for this bounding box.
+
+        Note: Here, ``[x1,y1]`` corresponds to the coordinates of the bottom-left corner of the bounding box, while ``[x2,y2]`` corresponds to the coordinates of the top-right corner of the bounding box. The last column, pred_prob, represents the predicted probability that the bounding box contains an object of the class k.
+
+        For more information see the `MMDetection package <https://github.com/open-mmlab/mmdetection>`_ for an example object detection library that outputs predictions in the correct format.
+
+    auxiliary_inputs: optional
+        Auxiliary inputs to be used in the computation of counts.
+        The `auxiliary_inputs` can be computed using :py:func:`rank._get_valid_inputs_for_compute_scores <cleanlab.object_detection.rank._get_valid_inputs_for_compute_scores>`.
+        It is internally computed from the given `labels` and `predictions`.
+
+    class_names: optional
+        A dictionary mapping one-hot-encoded class labels back to their original class names in the format ``{"integer-label": "original-class-name"}``.
+
+    Returns
+    -------
+    bbox_sizes: Tuple[Dict[Any, List], Dict[Any, List]]
+        A tuple containing two dictionaries. Each maps each class label to a list of the sizes of annotated bounding boxes for that class in the label and prediction datasets, respectively.
+    """
+    if auxiliary_inputs is None:
+        auxiliary_inputs = _get_valid_inputs_for_compute_scores(ALPHA, labels, predictions)
+
+    labl_area: DefaultDict[Any, list] = collections.defaultdict(list)
+    pred_area: DefaultDict[Any, list] = collections.defaultdict(list)
+    for sample in auxiliary_inputs:
+        _get_bbox_areas(sample["lab_labels"], sample["lab_bboxes"], labl_area, class_names)
+        _get_bbox_areas(sample["pred_labels"], sample["pred_bboxes"], pred_area, class_names)
+
+    return labl_area, pred_area
+
+
+def class_label_distribution(
+    labels=None,
+    predictions=None,
+    *,
+    auxiliary_inputs=None,
+    class_names: Optional[Dict[Any, Any]] = None,
+) -> Tuple[Dict[Any, float], Dict[Any, float]]:
+    """Returns the distribution of class labels associated with all annotated bounding boxes (or predicted bounding boxes) in the dataset.
+
+    This method can help you understand which classes are: rare or over/under-predicted by the model overall.
+
+    Parameters
+    ----------
+    labels:
```

Comment:
Can you hyperlink the format of labels/predictions/auxillary inputs for all functions but one in this page similarly to what is done [here](https://github.com/cleanlab/cleanlab/blob/6d45971cf83ccc85ac31d2591c54ce6b52a27281/cleanlab/object_detection/rank.py#L648)? (This way if we change anything in the format the update will only have to happen once per file)

I suggest linking labels/predictions definitions of `bounding_box_size_distribution` and `class_label_distribution` to the descriptions written out in `object_counts_per_image` since that appears first.